premake5.lua

```diff
@@ -211,7 +211,7 @@ solution "vaultClient"
 		filter { "kind:*App" }
 			targetextension ".js"
 		filter { "files:**.cpp" }
-			buildoptions { "-std=c++11" }
+			buildoptions { "-std=c++14" }
```

Comment:
Why was this change necessary?

Reply:
Yes, `udCallback` uses a C++14 feature, without this change it won't compile.